src/components/Registration/index.jsx

```diff
@@ -16,7 +16,7 @@ function Registration() {
 
   const submitHandler = (e) => {
     e.preventDefault();
-    axios.post('http://localhost:5050/users', users);
+    axios.post('http://localhost:5050/auth/signup', users);
```

Comment:
Don't use a hardcoded "http://localhost", it won't work in production
Time to use Dotenv !